locale/es/about/resources.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: about.hbs
+title: Logos y Gráficos
+---
+# Recursos
+
+## Descargas de Logo
+
+ Por favor revise la [Política de Marcas Registradas](/about/trademark/) para obtener información sobre el uso permitido de Node.js&reg; logos y marcas.
+
+ Las pautas para la visualización de la marca Node.js se describen en
+ la [Pautas Visuales](/static/documents/foundation-visual-guidelines.pdf).
+
+<table border="0" cellspacing="0" cellpadding="10" class="logos">
+  <tr>
+    <td bgcolor="#FFFFFF"><a href="/static/images/logos/nodejs-new-pantone-black.ai"><img src="/static/images/logos/nodejs-new-pantone-black.png" alt="Node.js on light background"></a></td>
```

Comment:
It would be nice to also translate images 'alt' atributes.

Reply:
thanks to you! :)